pom.xml

```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <artifactId>owasp-java-html-sanitizer</artifactId>
-  <packaging>jar</packaging>
+  <packaging>bundle</packaging>
```

Comment:
It'd be nice if this also happened to html-types/pom.xml and aggregate/pom.xml.

Reply:
I've also added the bundle packaging to the html-types/pom.xml. The aggregate/pom.xml doesn't need updated as it will still use both the root pom and html-types/pom to build bundles.